assemble/bin/accumulo-cluster

```diff
@@ -141,7 +141,7 @@ function control_service() {
     ACCUMULO_SERVICE_INSTANCE=""
     [[ $service == "tserver" && ${NUM_TSERVERS:-1} -gt 1 ]] && ACCUMULO_SERVICE_INSTANCE=${inst_id}
     [[ $service == "compactor" ]] && ACCUMULO_SERVICE_INSTANCE="${inst_id}_${5}"
-    [[ $service == "sserver" && ${NUM_SSERVERS:-1} -gt 1 ]] && ACCUMULO_SERVICE_INSTANCE=${inst_id}
+    [[ $service == "sserver" ]] && ACCUMULO_SERVICE_INSTANCE="${inst_id}_${5}"
```

Comment:
We could simplify the code a bit more and only use the `sserver1_<group>` or `compactor1_<queue>` patterns when there's more than one of each type. 

```
    ACCUMULO_SERVICE_INSTANCE=""
    # Only increment the service name when more than one service is desired.
    [[ last_instance_id -gt 1 ]] && ACCUMULO_SERVICE_INSTANCE="${inst_id}"
    [[ $service == "compactor" || $service == "sserver" ]] && ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}_${5}"
```

Reply:
> We could simplify the code a bit more and only use the sserver1_<group> or compactor1_<queue> patterns when there's more than one of each type.

There is one thing I like about always adding the `1` even if there is only a single server and that is the log naming.  If I start w/ a single server process then I will get a log file w/o the `1`.  If I change config to running 2 server process then I will get two logs files with numbers and the log file w/o a number is abandoned. 

I will take a stab at simplification.

Reply:
@ddanielr I decided to go ahead and merge this for now because we have both tested this change and did not want to test again after trying to make the code shorter.